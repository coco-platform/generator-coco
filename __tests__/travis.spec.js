/**
 * @description - coco:travis generator
 * @author - huang.jian <hjj491229492@hotmail.com>
 */

// packages
const path = require('path');
const fs = require('fs');
const renderer = require('yeoman-test');
const assert = require('yeoman-assert');

describe('coco:travis', () => {
  it('should generate travis configuration', () => {
    const travis = path.resolve(__dirname, '../travis');

    return renderer.run(travis).then(() => {
      assert.file(['.travis.yml']);

      const configuration = fs.readFileSync('.travis.yml', 'utf8');

      expect(configuration).toMatchSnapshot();
    });
  });
});
