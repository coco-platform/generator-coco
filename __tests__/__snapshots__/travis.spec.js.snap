// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`coco:travis should generate travis configuration 1`] = `
"sudo: false
language: node_js
cache:
  yarn: true
node_js:
  - 10
after_success:
  - cat ./coverage/lcov.info | ./node_modules/.bin/coveralls
"
`;
